### ISSUE TITLE
Voiceovers don't continue playing in English

### ISSUE THREAD

**Issue by julietugo** [oppia/oppia-android#4337]
Describe the bug
Talkback is not reading the English Language 

To Reproduce

1. Login to Oppia as Admin
2. Get to Fraction, Lessons
3. Go to Chapter One
4. Enable talkback
5. Navigate page
6. See error (watch video from 8.50 - 9.10)

Expected behavior
Talkback should continue to read the English Language instead of a different language

Actual Result: Talkback is reading Portuguese and another language.

Video link: https://drive.google.com/file/d/1T9vG_F27OEnul2sdhBD5qKiV1_JQjyrP/view?usp=sharing

Environment
Device name: Umidigi A3 Pro 
Android version: Android 9
App version : 0.6-alpha-de345f92ee

**Comment by BenHenning** [comment]
Thanks for filing this @julietugo! This looks to be another example of #3088 where voiceovers don't continue in English if one of the states is missing English voiceovers.

Marking as a duplicate, but it's helpful to have other examples of this occurring.

**Comment by BenHenning** [comment]
For searching context, this issue was found in 0.6-alpha (MR3).
